keps/sig-auth/5018-dra-adminaccess/README.md

```diff
@@ -541,7 +541,12 @@ rollout. Similarly, consider large clusters and how enablement/disablement
 will rollout across nodes.
 -->
 
-Will be considered for beta.
+- kube-controller-manager: If the kube-controller-manager fails to create `ResourceClaim` objects from `ResourceClaimTemplate` due to misconfigurations or permission issues relating to `adminAccess`, then the associated Pods will remain in a pending state and won't be scheduled.
+- kube-scheduler: Bugs in the scheduler might lead to Pods not being scheduled even when resources are available or, scheduling Pods that shouldn't be scheduled due to unmet `adminAccess` requirements. If the `DRAAdminAccess` feature gate isn't enabled or is misconfigured, the scheduler might not recognize ResourceClaim requirements, leading to scheduling failures.
```

Comment:
is this thinking of something more than generic scheduler backoff behavior when it encounters failed API requests?

Reply:
No, this should be part of the generic scheduler backoff behavior.

Reply:
ok, maybe clarify that... otherwise this line sounds scarier or more specific to this feature than it actually is